### DISCUSSION TITLE
Plan to add a second(maybe community theme?) as a alternative option?

### DISCUSSION THREAD

**Discussion by quank123wip** [Milkdown/milkdown#365] [Ideas]
As we can see, the author of this proejct had put much efforts on theme support, and the theme-nord is a good example of this. The problem is we don't have many community or third-party themes. It would be great to support a second theme in the official project as a secondary choice for new users, also a good practice for both the community and project contributors to do the thing, make a theme and make the theme stable(or prove theme's maintainability),

This may be a long-term plan and probably shouldn't do it now. If the project is ready to accept it, and I have my time, I'm glad to do this job.🤝♥（Unfortunately I'm a students now preparing for tests, it's hard to say when I can have my time.）

**Comment by Saul-Mirone** [comment]
Currently I'm fousing on Milkdown@v6, I plan to optimize the stylesheet organization in the version. So it's not a good time to create theme since there will be a breaking change for sure. I recommend you to wait for the realeasing of milkdown@v6 before starting.

**Comment by quank123wip** [comment]
> Currently I'm fousing on Milkdown@v6, I plan to optimize the stylesheet organization in the version. So it's not a good time to create theme since there will be a breaking change for sure. I recommend you to wait for the realeasing of milkdown@v6 before starting.

Great. I will keep on track and waiting for it. Thanks for your efforts on it. 😁
